### PR TITLE
Issue/unicode character in event titles causes an error

### PR DESCRIPTION
**Pull request by fmarco**

### CODE REVIEW

File: aldryn_events/tests/test_events.py [hunk 1]
```diff
@@ -186,6 +186,20 @@ def test_create_event(self):
...
+        with force_language('de'):
+            event = Event.objects.create(
+                title=test_title, slug='sprachgefuhl',
+                start_date=tz_datetime(2014, 9, 10),
+                publish_at=tz_datetime(2014, 9, 9),
+                app_config=self.app_config
+            )
+        self.assertEqual(Event.objects.translated('de').count(), 1)
+        self.assertEqual(test_title, event.title)
+        self.assertIn(test_title, unicode(event))
```
Comment (cyberstar) [inline aldryn_events/tests/test_events.py]: was this test case failed before the fix ?

Comment (fmarco) [inline aldryn_events/tests/test_events.py]: @cyberstar yes, this is the traceback FYI:

ERROR: test_specific_language_characters_in_title (aldryn_events.tests.test_events.EventTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "./aldryn_events/tests/test_events.py", line 201, in test_specific_language_characters_in_title
    self.assertIn(test_title, unicode(event))
  File "./aldryn_events/models.py", line 185, in __unicode__
    self.get_title(), app_config_namespace))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xfc' in position 9: ordinal not in range(128)

Comment (cyberstar) [inline aldryn_events/tests/test_events.py]: Cool thanks!

**Comment by cyberstar** [comment]
LGTM :+1:

**Pull request merged**